src/features/dashboard/components/ColumnRow/ColumnRow.module.css

```
@@ -0,0 +1,252 @@
+/* Column row (horizontal chip strip) */
+.columnRow {
+	display: flex;
+	flex-wrap: wrap;
+	align-content: flex-start;
+	align-items: center;
+	gap: 0.5rem;
+	/* 最多顯示約四列 chip（4 列高 + 3 個列間距），超過則於 chip 區內捲動*/
```

Stylelint (17.13.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)